sites/fast-frame/src/public/index.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
Seems I can't comment on an empty file but whats the reason the `app/.index` is a .js file?